modules/client/examples/04-addresslistVoting-client/06-can-vote.ts

```diff
@@ -0,0 +1,33 @@
+/* MARKDOWN
+### Check if an address can vote in a proposal (Addresslist)
+
+Checks whether an address is able to participate in a DAO proposal created using the Addresslist Voting plugin.
+*/
+
+import {
+  AddresslistVotingClient,
+  ContextPlugin,
+  ICanVoteParams
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the aragonOSx SDK context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an AddresslistVoting client.
+const addresslistVotingClient = new AddresslistVotingClient(contextPlugin);
+
+const voteParams: ICanVoteParams = {
+  address: "0x1234567890123456789012345678901234567890",
```

Comment:
@josemarinas this is inconsistent with the `addressOrEns` above. 

To avoid confusions, I would name all of them like `targetAddress`. If there's an ENS, we silently resolve it